packages/web/src/views/governance/components/my-delegation/MyDelegation.tsx

```diff
@@ -0,0 +1,177 @@
+
+import InfoBox from "../info-box/InfoBox";
+
+import TokenChip from "../token-chip/TokenChip";
+import {
+  MyDelegationTooltipContent,
+  MyDelegationWrapper,
+} from "./MyDelegation.styles";
+import MissingLogo from "@components/common/missing-logo/MissingLogo";
+
+interface MyDelegationProps {
+  myDelegationInfo: MyDelegationInfo;
+  isLoading: boolean;
+}
+
+const MyDelegation: React.FC<MyDelegationProps> = ({
+  myDelegationInfo,
+  isLoading,
+}) => {
+  const { t } = useTranslation(); 
+  
+  const [showUndel, setShowUndel] = useState(false);
+
+  const hasUndel = !!myDelegationInfo.undeligatedAmount;
+
+  const votingWeightInfos = myDelegationInfo.delegations.filter(
+    item => !item.unlockDate,
+  );
+  const undeligationInfos = myDelegationInfo.delegations.filter(
+    item => item.unlockDate,
+  );
+
+  return (
+    <MyDelegationWrapper>
+      <div className="my-delegation-title">{t("Governance:myDel.title")}</div>
+      <div className="info-wrapper">
+        <InfoBox
+          title={t("Governance:myDel.availBal.title")}
+          value={
+            <>
+              {myDelegationInfo.availableBalance.toLocaleString("en")}
+              <TokenChip tokenInfo={GNS_TOKEN} />
+            </>
+          }
+          tooltip={t("Governance:myDel.availBal.tooltip")}
+          isLoading={isLoading}
+        />
+        <InfoBox
+          title={
+            hasUndel && showUndel
+              ? t("Governance:myDel.undel.title")
+              : t("Governance:myDel.votingWeight.title")
+          }
+          value={
+            <Tooltip
+              FloatingContent={
+                <MyDelegationTooltipContent>
+                  {(showUndel ? undeligationInfos : votingWeightInfos).map(
+                    (item, index) => (
+                      <div
+                        key={`del-item-${item.updatedDate}-${index}`}
+                        className="delegation-item"
+                      >
+                        {index !== 0 && <div className="divider" />}
+                        <div className="info-row">
+                          <div className="info-subject">
+                            {t("Governance:myDel.tooltip.delegatee")}
+                          </div>
+                          <div className="info-value">
+                            <MissingLogo
+                              symbol={item.delegatee}
+                              url={item.logoUrl}
+                              width={20}
+                            />
+                            {item.delegatee}
+                          </div>
+                        </div>
+                        <div className="info-row">
+                          <div className="info-subject">
+                            {t("Governance:myDel.tooltip.amount")}
+                          </div>
+                          <div className="info-value">
+                            {item.amount.toLocaleString("en")} GNS
```

Comment:
What about using the Symbol of a variable in a `GNS_TOKEN`?

Reply:
fix it in 61a66d30445fa39734e92b3c747396313864b011